src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/DataFrameTests.cs

```diff
@@ -266,36 +346,26 @@ public void TestGroupedMapUdf()
             }
         }
 
-        private static RecordBatch CountCharacters(RecordBatch records)
+        private static FxDataFrame CountCharacters(FxDataFrame dataFrame, string groupFieldName, string stringFieldName)

```

Comment:
Do we still want the RecordBatch test as well? Might as well test it, as long as it is supported.

Reply:
Yup, good catch. I must've missed it in my review. I'll add it.

Reply:
Yup, good catch. I must've missed it in my review. Added